packages/@aws-cdk/aws-iot-actions/lib/sqs-queue-action.ts

```diff
@@ -0,0 +1,54 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as iot from '@aws-cdk/aws-iot';
+import * as sqs from '@aws-cdk/aws-sqs';
+import { CommonActionProps } from './common-action-props';
+import { singletonActionRole } from './private/role';
+
+/**
+ * Configuration properties of an action for s3.
+ */
+export interface SQSQueueActionProps extends CommonActionProps {
```

Comment:
We camel-case acronyms in CDK:

```suggestion
export interface SqsQueueActionProps extends CommonActionProps {
```

Reply:
Thank you. I fixed.